### DISCUSSION TITLE
Global tone mapping....Hero 11 Black

### DISCUSSION THREAD

**Discussion by BradACI** [gopro/labs#296] [General]
What are your thoughts about setting tone mapping to global

**Comment by dnewman-gpsw** [comment]
I don't generally have to worry about that control, and using Color Flat or any other LOGB (logbase) log curve, disables all tone-mapping.  GTM is just a scene dependent contrast the camera can add, I prefer that to add contrast (as needed) in post.

**Reply by BradACI** [reply]
Good deal. Flat is what I use all the time. Right on

**Reply by andrerav** [reply]
If you use Davinci Resolve, I can highly recommend trying the "contrast pop" plugin for working with local contrast. I have found that creating three parallel nodes with separate contrast pop for highlights, mid/shadow and global can create really nice results. This is particularly useful in scenes with large bright areas such as snow or white houses where you want to bring out more texture in the snow without overdoing the contrast in already well defined areas of the scene. Here's an example from an edit I was working on last night:

![image](https://user-images.githubusercontent.com/3635018/230579983-4c40febb-2d48-4524-b1d6-7cc7ee42056a.png)

Note that the contrast pop might be a paid feature that only comes with Resolve Studio (which is what I'm using).
